void tomo::IoLoop::run_loop<true, false, false, true, true>()
--- PRE

+++ POST

@@ -597,7 +597,7 @@

 0b4c [ 4] nopl 0x0(%rax)
 0b50 [ 5] mov $0x1,%esi
 0b55 [ 3] mov %rbx,%rdi
-0b58 [ 5] call <tomo::IoLoop::arm_accept(tomo::UrKind)>
+0b58 [ 5] call <tomo::IoLoop::fliplisten_retry_accept(tomo::UrKind)>
 0b5d [ 5] jmp <+0x2e2>
 0b62 [11] data16 cs nopw 0x0(%rax,%rax,1)
 0b6d [ 3] nopl (%rax)
